### PR TITLE
SLURM migration

### PR DESCRIPTION
**Pull request by pmb59**

### CODE REVIEW

**Review by anilthanki — APPROVED** [review]
LGTM

File: nextflow.config [hunk 1]
```diff
@@ -1,20 +1,21 @@
...
-    executor='lsf'
-    
-    withName: 'download_fastqs' {
-        clusterOptions = "-g /scxa_ena_download"
-    }
-}
-
-executor {
+    executor='slurm'
+    queue="$SCXA_HPC_QUEUE"
```
Comment (irisdianauy) [inline nextflow.config:3]: A small suggestion: Instead of `queue` we can use `partition` since that's the equivalent in SLURM.

**Review by irisdianauy — APPROVED** [review]
If `SCXA_HPC_QUEUE` and `SCXA_HPC_OPTIONS` are defined in the running env, this looks good. Also added a small suggestion re `queue`.  For future, we can move away from hardcoding `time` in the code, as they can change the max time in the partition we are using.

File: nextflow.config (hunk 1 above)
Comment (pmb59) [reply nextflow.config:3]: @irisdianauy  We can't, the process directive `queue` is fixed by Nextflow https://www.nextflow.io/docs/latest/executor.html#slurm

**Comment by pmb59** [comment]
Thanks @irisdianauy and @anilthanki for reviewing!

**Pull request merged**